twittermap/conf/application.conf

```diff
@@ -78,8 +78,16 @@ bounded-mailbox {
 
 cloudberry.register = "http://localhost:9000/admin/register"
 cloudberry.ws = "ws://localhost:9000/ws"
+
+# TwitterMap Extra Function Knobs
+# Sentiment Analysis
 sentimentEnabled = false
 sentimentUDF = "twitter.`snlp#getSentimentScore`(text)"
+# Customize TwitterMap for a specific domain
+removeSearchBar = true
```

Comment:
@HotLemonJuice can you make it `false` by default?  and then show some gif to see if the UI is the same as before?